docker/docker_official_images/3.7.0/jvm/jsa_launch

```diff
@@ -17,7 +17,7 @@
 KAFKA_CLUSTER_ID="$(opt/kafka/bin/kafka-storage.sh random-uuid)"
 TOPIC="test-topic"
 
-KAFKA_JVM_PERFORMANCE_OPTS="-XX:ArchiveClassesAtExit=storage.jsa" opt/kafka/bin/kafka-storage.sh format -t $KAFKA_CLUSTER_ID -c opt/kafka/config/kraft/reconfig-server.properties
+KAFKA_JVM_PERFORMANCE_OPTS="-XX:ArchiveClassesAtExit=storage.jsa" opt/kafka/bin/kafka-storage.sh format --standalone -t $KAFKA_CLUSTER_ID -c opt/kafka/config/kraft/reconfig-server.properties
```

Comment:
I think this is an error, In kafka 3.7.0 version, there doesn't have `--standalone` argument and doesn't have `reconfig-server.properties`
https://github.com/apache/kafka/blob/396c5d5d42baa9fd32cf9f38d6095f1133f300fb/core/src/main/scala/kafka/tools/StorageTool.scala#L149